packages/react-dom-bindings/src/client/ReactDOMFloatClient.js

```diff
@@ -223,6 +242,10 @@ function preinit(href: string, options: PreinitOptions) {
 
     switch (as) {
       case 'style': {
+        let styleResources = resourceRoot._rstyles;
+        if (!styleResources) {
+          styleResources = resourceRoot._rstyles = new Map();
```

Comment:
This should really be done in here: 

https://github.com/facebook/react/blob/9f8a98a390bca3310cb80f536ba015446af19c3f/packages/react-dom-bindings/src/client/ReactDOMComponentTree.js#L36-L44

So that we put all these hacks in one place.